### DISCUSSION TITLE
Simulation System

### DISCUSSION THREAD

**Discussion by nbeuttl** [camueller/SmartApplianceEnabler#492] [Allgemein]
Hallo zusammen,
Ich bekomme wahrscheinlich in 12 Monaten ein System mit Wechselrichter, SHM 2.0, Idm-WP mit intelligenter Schnittstelle, schaltbare Steckdosen, Wallbox und später eine Batterie.
Um die Zeit zu nutzen wäre es schön, wenn ich die Komponenten raspberry und SHM 2.0 real aufbauen, mit der nötigen Software versorgen (SAE, etc.), die PV-Leistung und die Verbraucher simulieren könnte.
Wäre so etwas denkbar?

**Comment by camueller** [comment]
Du kannst den SAE installieren und ihm dieselben Befehle schicken, wie es der SHM macht. Aber wozu soll das gut sein?

**Comment by nbeuttl** [comment]
Hallo Axel,So ne schnelle Reaktion habe ich nicht erwartet!  Meine Motivation ist folgende:Ich möchte mich vorab mit dem Thema Raspberry, SAE und SHM 2.O beschäftigen:- Design des Gesamtsystems - Kofiguration dieser Module- Ausserdem die Verbraucher in den Systemen konfigurieren- Das Handling erlernenUnd das in aller Ruhe und GelassenheitUm schnell das System, wenn es ernstwird, in Betrieb nehmen zu können.Sei herzlich gegrüßt Norbert--Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.--Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 05.03.23, 08:42 schrieb "Axel Müller" ***@***.***>:

   
  Du kannst den SAE installieren und ihm dieselben Befehle schicken, wie es der SHM macht. Aber wozu soll das gut sein?

**Comment by camueller** [comment]
> Ich möchte mich vorab mit dem Thema Raspberry, SAE und SHM 2.O beschäftigen

Ohne echte PV-Hardware wirst Du nur sehr wenig echt Erkenntnisse gewinnen und sowieso alles ändern müssen, wenn Du dann die echte PV-Hardware hast. Vielleicht ist es sinnvoller, sich um eine schnellere Installation der PV-Hardware zu bemühen.

**Comment by A124Bob** [comment]
Hallo Norbert,
ich habe ebenfalls die Kombination mit SHM 2.0 und IDM-Wärmepumpe mit Navigator 2.0.
Auch ShellyPlug und ShellyPM2.5.
Zusammen mit openWB läuft alles.
